src/opt/ImproveMemoryOps.cpp

```diff
@@ -1489,60 +1343,36 @@ static llvm::Instruction *lGSToLoadStore(llvm::CallInst *callInst) {
 ///////////////////////////////////////////////////////////////////////////
 // MaskedStoreOptPass
 
+static std::unordered_map<std::string, int> maskedStoreAlign = {
+    {__pseudo_masked_store_i8, 1},     {__pseudo_masked_store_i16, 2},   {__pseudo_masked_store_half, 2},
```

Comment:
It's not directly related to this PR but would be good to fix: the alignment values are used at least 4 times in this file. I suggest saving them into separate enum and using it instead of explicit integer values.

Reply:
`Alignment` enum is added